### ISSUE TITLE
Support for imported csproj files

### ISSUE THREAD

**Issue by maartenoosterhoff** [BMuuN/vsts-assemblyinfo-task#93]
Version 2 does not (yet) support imported (secondary) csproj files.

For example, here is the partial contents of my myProject.csproj file:

```
<Project Sdk="Microsoft.NET.Sdk">
  <Import Project="..\GlobalSettings.csproj" />
  <PropertyGroup>
```

And here is the partial contents of my GlobalSettings.csproj file:L

```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup>
    <AssemblyVersion>0.0.1.0</AssemblyVersion>
    <FileVersion>0.0.1.0</FileVersion>
    <Version>0.0.1-local-build</Version>
```

When the Azure build-pipeline runs, I get the following message in the task (which has a warning):

```
Setting .Net Core / .Net Standard assembly info...
Processing: D:\a\1\s\GlobalSettings.csproj
	Detected character encoding: utf-8
##[warning]	Project is not targeting .Net Core or .Net Standard, moving to next file.
```

This mechanism was supported in version 1. I hope that it will return in Version 2.

**Comment by IanMercer** [comment]
BTW [Directory.Build.Props](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019) is a nicer way to handle sharing settings across multiple projects, and there's a PR for supporting that.

**Comment by BMuuN** [comment]
This has been fixed with #117
